nss-day-cohort-52/rare-api-smoky-pythons
# Lz fixtures

### Issue ticket(s): #6 #16 #17 

### New file(s): None

### Modified File(s): views/category.py

### Modifications: Added a create, update, and destroy method to categories. 

### Steps to test:
1. While in the terminal, use git fetch --all in the root directory of this repo
2. Checkout out to lz-fixtures
3. Open postman and check to see if you can create a category, edit a category, and delete a category.
